src/hypothesis/settings.py

```diff
@@ -278,7 +278,7 @@ def __exit__(self, *args, **kwargs):
 
 Settings.define_setting(
     u'max_examples',
-    default=200,
+    default=int(os.getenv(u'HYPOTHESIS_MAX_EXAMPLES', 200)),
```

Comment:
Still have to wrap it in int. When I have the var set in my shell I get this
```
>>> os.environ.get(u'HYPOTHESIS_MAX_EXAMPLES', 203)
'200'
```

As for `os.getenv` vs `os.environ.get` I agree but was following the pattern of the file.